task/sast-coverity-check/0.2/sast-coverity-check.yaml

```diff
@@ -162,7 +162,6 @@ spec:
   stepTemplate:
     computeResources:
       limits:
-        cpu: "4"
```

Comment:
This change was already merged via #1877 and does not cause any conflict with this pull request.

Reply:
Rebased my PR, so this change has been included.. `build` step was missing the removing cpu limit, made the change as part of this PR.